.github/workflows/autodocs-platform.yaml

```diff
@@ -7,78 +7,15 @@ on:
   push:
     branches:
       - platform-docs
-env:
-  PLATFORM_CHANGELOG: "autodocs/platform-changelog.md"
 
 jobs:
-  check-new-docs:
-    runs-on: ubuntu-latest
-
-    if: github.repository == 'chainguard-dev/edu'
```

Comment:
Last minute thing, I'm going to put this into the `integrate-platform-docs` job below.  Don't want forks running this workflow